io.github.jazztickets.irrlamb.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.jazztickets.irrlamb</id>
+	<name>irrlamb</name>
+	<summary>3D physics game in which you control a blue ball of energy in an attempt to touch all glowing orbs</summary>
+	<description>
+		<p>Find all the orbs in this challenging 3D physics game, using either mouse/keyboard or gamepad.</p>
+		<p>35 levels, including 15 tutorials, 7 "campaign" levels, 4 skateparks, 7 challenges, and 2 secrets.</p>
+		<p>Keep track of your fastest times and save replays.</p>
+	</description>
+	<categories>
+		<category>Game</category>
+		<category>ArcadeGame</category>
+	</categories>
+	<content_rating/>
```

Comment:
this is likely invalid.

Reply:
I ran it with `flatpak run org.freedesktop.appstream-glib validate io.github.jazztickets.irrlamb.appdata.xml` and it says OK. I'll change it to `<content_rating type="oars-1.1" />`